Mathlib.lean

```diff
@@ -194,6 +196,7 @@ import Mathlib.Algebra.Group.Hom.Basic
 import Mathlib.Algebra.Group.Hom.CompTypeclasses
 import Mathlib.Algebra.Group.Hom.Defs
 import Mathlib.Algebra.Group.Hom.Instances
+import Mathlib.Algebra.Group.Hom.Ring
```

Comment:
Similarly, this file name looks weird. Could it either be `...Group.Hom.End` or `...Ring.Hom.something`? Semantically I prefer the former. In terms of library stratification, I prefer the latter.

Reply:
Happy if you'd like to push a rename.